packages/codegen-ui/lib/generate-form-definition/form-to-component.ts

```diff
@@ -13,66 +13,86 @@
-        name: `${fieldName}Field`,
-        componentType: 'TextField',
-        properties: {
-          name: {
-            value: fieldName,
-          },
-          label: {
-            value: fieldName,
-          },
-          placeholder: {
-            value: `${fieldValue.type}`,
-          },
-          ...(fieldValue.isRequired && {
-            required: {
-              value: 'true',
-              type: 'boolean',
-            },
-          }),
-        },
-      });
-    }
-  });
+const DefaultStudioFormStyles = {
+  horizontalGap: '15px',
+  verticalGap: '15px',
+  outerPadding: '20px',
+  alignment: 'left',
+};
 
-  return formFields;
+const getStyleResolvedValue = (config?: FormStyleConfig): string | undefined => {
+  return config?.value ?? config?.tokenReference;
 };
 
-export const mapParentGrid = (name: string, children: StudioComponentChild[] = []): StudioComponentChild => {
+export const resolveStyles = (style: StudioFormStyle): Record<keyof Omit<StudioFormStyle, 'alignment'>, string> => {
   return {
-    name: `${name}Grid`,
+    verticalGap: getStyleResolvedValue(style.verticalGap) ?? DefaultStudioFormStyles.verticalGap,
+    horizontalGap: getStyleResolvedValue(style.horizontalGap) ?? DefaultStudioFormStyles.horizontalGap,
+    outerPadding: getStyleResolvedValue(style.outerPadding) ?? DefaultStudioFormStyles.outerPadding,
+  };
+};
+
+export const parentGrid = (
+  name: string,
+  style: StudioFormStyle,
+  children: StudioComponentChild[],
+): StudioComponentChild => {
+  const { verticalGap, horizontalGap } = resolveStyles(style);
+  return {
+    name,
     componentType: 'Grid',
     properties: {
       columnGap: {
-        value: '1rem',
+        value: verticalGap,
       },
       rowGap: {
-        value: '1rem',
+        value: horizontalGap,
       },
     },
     children,
   };
 };
 
+export const fieldComponentMapper = (name: string, formDefinition: FormDefinition): StudioComponentChild => {
+  // will accept a field matrix from a defnition and map
+  const fieldChildren = formDefinition.elementMatrix.map<StudioComponentChild>((row: string[], rowIdx: number) => {
+    return {
+      name: `RowGrid${rowIdx}`,
+      componentType: 'Grid',
+      properties: {
+        columnGap: { value: 'inherit' },
+        rowGap: { value: 'inherit' },
+        ...(row.length > 0 && {
+          templateColumns: { value: `repeat(${row.length}, 1fr)` },
+        }),
+      },
+      children: row.map<StudioComponentChild>((column, colIdx) => {
+        const element: FormDefinitionElement = formDefinition.elements[column];
+        return {
+          name: `${element.componentType}${colIdx}`,
+          componentType: element.componentType,
+          properties: Object.entries(element.props).reduce((prev: StudioComponentProperties, [propKey, propValue]) => {
+            // eslint-disable-next-line no-param-reassign
+            prev[propKey] = { value: `${propValue}`, type: `${typeof propValue}` };
```

Comment:
Yes, I was wondering if it needs to be "Boolean" instead of "boolean" for codegen to render the right Typescript. I thought I saw a helper that wants capitalized types. Resolving for now. I'm sure we'll update this later if we need to.